bitnami/keycloak/values.yaml

```diff
@@ -630,9 +630,11 @@ service:
   ##
   type: LoadBalancer
   ## @param service.port Service HTTP port
+  ## 0 will disable http-mapping
```

Comment:
If you want this to appear in the README it has to be added in the line above. Same for the next one